encode/h264encode.c

```diff
@@ -1657,17 +1657,17 @@ static int render_packedpicture(void)
     return 0;
 }
 
-static void render_packedsei(void)
+/*static void render_packedsei(void)
```

Comment:
Is this (as well as at 1723) going to be removed? Why are they commented out?

Reply:
It was dead code. Removed.